src/main.cc

```diff
@@ -3,20 +3,19 @@
 
 #include "gmsh.h"
 #include "io.h"
-#include "json.hpp"
 
 int main(int argc, char** argv) {
   try {
     if (argc != 3) {
       std::cout << "Usage: ./mpm-point-generator /path/to/path_file.txt\n"
-                   "Dimenssion 2 or 3\n";
+                   "Dimension 2 or 3\n";
       throw std::runtime_error("Incorrect number of input arguments");
     }
-    //! Get dimension or Tdim
-    int dimension = std::atoi(argv[2]);
+    //! Get Tdim
+    int Tdim = std::atoi(argv[2]);
```

Comment:
Yes, should be `const`. I am sorry. and yes, it's template, so I used `Tdim` for the name. Do you want `ndimension` as the name?